org.codeberg.SHuRiKeN.Eeman.json

```diff
@@ -0,0 +1,100 @@
+{
+    "id" : "sh.shuriken.Eeman",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "45",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "eeman",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--socket=session-bus",
```

Comment:
`session-bus` is not allowed.

Reply:
The thing is I am unable to figure out which session-bus is the application using.. so that is an issue.

Reply:
--log-session-bus passed to `flatpak run` should gives some idea.